### DISCUSSION TITLE
Sloped Railings

### DISCUSSION THREAD

**Discussion by chrismilne300** [OpenRCT2/OpenRCT2#21568] [Feature requests] [answered]
Almost every other type of fence or wall can be placed on a slope except the railings that I believe were added in vanilla RCT2. I'm sure it would be possible to add a sloped sprit to these railings.
![Untitled-1](https://github.com/OpenRCT2/OpenRCT2/assets/1381723/755715f4-c45f-47ca-a4ee-f3a92db24804)

**Comment by ZeeMaji** [comment, marked as answer]
This was just implemented in the objects repo a few days ago, it's not available in develop yet but it will be in the near future.

Slope sprites are also being added to other wall types too beyond just these.
